spec/message_repository_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'message_repository'
+require 'message'
+
+def reset_messages_table
+    seed_sql = File.read('spec/seeds/seeds_chitter_challenge_test.sql')
+    connection = PG.connect({ host: '127.0.0.1', dbname: 'chitter_challenge_test' })
+    connection.exec(seed_sql)
+end
+
+RSpec.describe MessageRepository do
+
+    before(:each) do 
+        reset_messages_table
+    end
+
+    context 'testing the MessageRepository class' do
+        it 'return all messages' do
+            repo = MessageRepository.new
+            message = repo.all
+
+            expect(message.length).to eq 3 
+            expect(message[0].id).to eq 1 
+            expect(message[0].title).to eq 'paired programming' 
+            expect(message[0].content).to eq 'learning to pair program'
+            expect(message[0].tags).to eq '@mat'
+        end
+
+        it 'create a new message' do
```

Comment:
if you word this as 'creates a new message', it just reads slightly better when running the tests